tasks/eva_3399/src/main/groovy/eva3399/RemediateIndels.groovy

```diff
@@ -0,0 +1,249 @@
+package eva3399
+
+import eva3399.ClashingSSHashes
+import org.slf4j.LoggerFactory
+import org.springframework.data.mongodb.core.query.Update
+import uk.ac.ebi.ampt2d.commons.accession.core.models.EventType
+import uk.ac.ebi.eva.accession.core.EVAObjectModelUtils
+import uk.ac.ebi.eva.accession.core.model.dbsnp.DbsnpSubmittedVariantOperationEntity
+import uk.ac.ebi.eva.accession.core.model.eva.SubmittedVariantEntity
+import uk.ac.ebi.eva.accession.core.model.eva.SubmittedVariantInactiveEntity
+import uk.ac.ebi.eva.accession.core.model.eva.SubmittedVariantOperationEntity
+import uk.ac.ebi.eva.groovy.commons.EVADatabaseEnvironment
+import uk.ac.ebi.eva.groovy.commons.RetryableBatchingCursor
+import uk.ac.ebi.eva.remapping.ingest.batch.io.SubmittedVariantDiscardPolicy
+
+import static org.springframework.data.mongodb.core.query.Criteria.where
+import static org.springframework.data.mongodb.core.query.Query.query
+import static uk.ac.ebi.eva.groovy.commons.EVADatabaseEnvironment.*
+import static uk.ac.ebi.eva.remapping.ingest.batch.io.SubmittedVariantDiscardPolicy.SubmittedVariantDiscardDeterminants
+
+class RemediateIndels {
+    static def logger = LoggerFactory.getLogger(RemediateIndels.class)
+    String assembly
+    EVADatabaseEnvironment prodEnv
+    EVADatabaseEnvironment devEnv
+    boolean targetAssembly
```

Comment:
Might prefer calling this `isTargetAssembly`, on first read I thought it was an assembly accession rather than a boolean...

Reply:
Will address..